js/about/contributionStatement.js

```diff
@@ -26,7 +26,10 @@ const ipc = window.chrome.ipcRenderer
 const {StyleSheet, css} = require('aphrodite/no-important')
 const globalStyles = require('../../app/renderer/components/styles/global')
 
-class ContributionStatement extends ImmutableComponent {
+const braveLogo = require('../../app/extensions/brave/img/braveAbout.png')
+const verifiedIcon = require('../../app/extensions/brave/img/ledger/verified_green_icon.svg')
+
+class ContributionStatement extends React.Component {
```

Comment:
@willy-b looks like this is now using React's `setState()`. Our `ImmutableComponent` wrapper will only trigger a re-render if properties are updated (not state).  Changing this forces the setState() to re-render